### DISCUSSION TITLE
A dynamic progress bar

### DISCUSSION THREAD

**Discussion by SoumyaK4** [cotes2020/jekyll-theme-chirpy#2100] [Q&A] [answered]
### Description

I really love this [progress.js](https://github.com/kungfux/kungfux.github.io/blob/main/assets/js/progress.js) by @kungfux 
Hope to see it included in Chirpy in future.

**Comment by kungfux** [comment]
@SoumyaK4 It will not be a part of this repo per #1771.

The good news is that you can integrate it into your blog yourself.

**Comment by SoumyaK4** [comment, marked as answer]
> @SoumyaK4 It will not be a part of this repo per #1771.
> 
> The good news is that you can integrate it into your blog yourself.

Nice I'll be using it 😁 
Will keep `watching` your repo for stuff like this ❤️
